app/models/concerns/inventory_computer.rb

```diff
@@ -90,6 +103,18 @@ def update_package_quantities!(package)
           package.save!
         end
       end
+
+      private
+
+      def box_pallet_on_hand_qty_hash(value)
```

Comment:
what's `value` here ?

Reply:
changed it to a better name now. have a look